specs/registry-auth.md

```diff
@@ -95,7 +95,7 @@ Notation follows the [Docker Registry v2 authentication][oauth2] specification f
 
 ## Credential Store
 
-As local credentials may be required to access the remote registries, they need to be stored and accessed securely. To achieve maximum security, credential helpers are preferred so that credentials are stored in the system key chain with better protection. If credential helpers are not available, notation will fall back to credential files with proper access control.
+As local credentials may be required to access the remote registries, they need to be stored and accessed securely. To achieve maximum security, credential helpers are preferred so that credentials are stored in the system key chain with better protection. If credential helpers are not available, credentials SHOULD be provided to notation via command line parameters `--username` / `--password` or environment variables `NOTATION_USERNAME` / `NOTATION_PASSWORD`.

```

Comment:
username/password options are only supported in `notation loging` cmd now. Should we mention it here?

Reply:
Yes, we need to clarify what is or is not supported.

Reply:
username/password options are also supported for `notation sign`, `notation verify`, and other commands which need registry accesses.